support/k8s/k8s-workload-registrar/webhookhandler.go

```diff
@@ -38,25 +117,78 @@ func (h *WebhookHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	var in admv1beta1.AdmissionReview
-	if err := json.NewDecoder(req.Body).Decode(&in); err != nil {
-		http.Error(w, "Malformed JSON body", http.StatusBadRequest)
+	var body []byte
+	if req.Body != nil {
```

Comment:
This check is superfluous and IMO a bit overly-defensive. `req.Body` will not be nil when this handler is invoked.

Reply:
yes you are right, I removed this control, fixed it, thank you 🙋🏻‍♂️🤝